code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -173,4 +173,25 @@
 		/obj/item/weapon/reagent_containers/spray,
 		/obj/item/weapon/soap,
 		/obj/item/weapon/holosign_creator
+		)
+
+/obj/item/weapon/storage/belt/bandolier
+	name = "bandolier"
+	desc = "A bandolier for holding shotgun ammunition."
+	icon_state = "bandolier"
+	item_state = "bandolier"
+	storage_slots = 6
+	can_hold = list(
+		/obj/item/ammo_casing/shotgun,
+		/obj/item/ammo_casing/shotgun/buckshot,
+		/obj/item/ammo_casing/shotgun/beanbag,
+		/obj/item/ammo_casing/shotgun/improvised,
+		/obj/item/ammo_casing/shotgun/stunslug,
+		/obj/item/ammo_casing/shotgun/meteorshot,
+		/obj/item/ammo_casing/shotgun/pulseslug,
+		/obj/item/ammo_casing/shotgun/incendiary,
+		/obj/item/ammo_casing/shotgun/frag12,
+		/obj/item/ammo_casing/shotgun/incendiary/dragonsbreath,
+		/obj/item/ammo_casing/shotgun/techshell,
+		/obj/item/ammo_casing/shotgun/dart

```

Comment:
...you don't have to put all of these ammo types in here, /obj/item/ammo_casing/shotgun already does that, they're children of it